code/pizzabot.js

```diff
@@ -1,7 +1,37 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian"
+const hawaiian = "Hawaiian"
+const pepperoni = "Pepperoni"
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+alert(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`)
+let orderName = prompt('Enter the name of the pizza you want to order today.');
 
-//Put your Javscript code here:
+const validateOrderName = () => { 
+    if (orderName.toLowerCase() === vegetarian.toLowerCase() || orderName.toLowerCase() === hawaiian.toLowerCase() || orderName.toLowerCase() === pepperoni.toLowerCase()) { 
+        orderQuantity = prompt(`How many ${orderName} pizzas do you want?`);
+        return true; 
```

Comment:
No need to add `return true` over here - your code is working perfectly fine without it. As mentioned already, if you are returning `boolean` value, you probably want to assign that value later on to some variable. But in your case it's not relevant